tools/extra/ras/main.c

```diff
@@ -909,14 +909,50 @@ fpga_result inject_ras_errors(fpga_handle fme_handle,
 	return result;
 }
 
+fpga_result clear_fpga_errors(fpga_handle handle, char *error )
+{
+	fpga_result result = FPGA_OK;
+	fpga_object error_object;
+	uint64_t value; 
+
+	if (!error)
+		return FPGA_INVALID_PARAM;
+
+	// Clear FME error
+	result = fpgaHandleGetObject(handle, error, &error_object, 0);
+	if (result != FPGA_OK) {
+		OPAE_ERR("Failed to get handle Object");
+		return result;
+	}
 
+	result = fpgaObjectRead64(error_object, &value, 0);
+	if (result != FPGA_OK) {
+		OPAE_ERR("Failed to Read Object ");
+		return result;
```

Comment:
memory leak here, we're not destroying `error_object`

Reply:
fixxed